### ISSUE TITLE
MDN navigator.id.get extra parameters

### ISSUE THREAD

**Issue by stenington** [mozilla/persona#2380]
https://developer.mozilla.org/en-US/docs/DOM/navigator.id.get

It seems from experimentation that `navigator.id.get` can take the same parameters as `navigator.id.request`, including `returnTo` and `siteName`. However these are not documented on the MDN page, making it look as if you have to use the Observer API to get those features.

**Comment by callahad** [comment]
Initially, the additional features were only planned for the Observer API, though an implementation quirk enabled them for `.get()` by mistake (#2086) and was resolved in pull request #2109. We've since decided to re-enable them, but haven't landed code for that yet.

Our next major deploy (early September) will disable these options with the `.get()` API. We'll re-enable them later that month (#2382).

In short: Don't rely on them yet unless you're using the Observer API. :)

**Comment by stenington** [comment]
Oh wow, ok. 

@callahad Will passing `returnTo` and `siteName` to `get()` cause errors, or will they just get ignored?

**Comment by callahad** [comment]
@stenington They'll just be ignored. Sorry for the confusion! Is there a reason you prefer `.get` over the Observer API, or were you primarily concerned about the experimental messaging on the Observer docs?

**Comment by stenington** [comment]
@callahad I think we're happy to move to the Observer API eventually, but yeah, the experimental messaging was a little worrisome for now.
